miden-tx/src/tests.rs

```diff
@@ -91,20 +111,20 @@ fn test_transaction_result_account_delta() {
             push.0 movdn.5 push.0 movdn.5 push.0 movdn.5
             # => [index, V', 0, 0, 0]
 
-            call.account_{account_id}::set_item
+            call.0xaa7d21ef60871f591cbb16b85907c6674c0a834740deab336c348c95269a747f
             # => [R', V]
         end
 
         proc.set_code
-            call.account_{account_id}::set_code
+            call.0x108b8fae888827e2c106af8567c308da9f8d8f9bb5c88a24c0ca6e394d73ffdb
             # => [0, 0, 0, 0]
 
             dropw
             # => []
         end
 
         proc.incr_nonce
-            call.account_{account_id}::incr_nonce
+            call.0xd765111e22479256e87a57eaf3a27479d19cc876c9a715ee6c262e0a0d47a2ac
```

Comment:
These hashes correspond to procedures on the account, right? Maybe we should create constants for them and then use constants here? This way, it would be very easy to tell what's going on.